matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/MatchStorageManager.java

```diff
@@ -41,9 +41,12 @@
      * @param phenScore only matches with phenotypical score higher or equal to this value are returned
      * @param genScore only matches with genotypical score higher or equal to this value are returned
      * @param onlyCurrentUserAccessible when true only matches that current user has access to are returned
+     * @param fromDate only matches found on or after the specified date (0 by default)
+     * @param toDate only matches found on or before the specified date (current date by default)
```

Comment:
These are not correct, and not informative enough. A `fromDate` of `0` doesn't make sense, since there's no such date, and there's actually no default value for them, since Java doesn't have default values for parameters. There is a default behavior for `null` or empty strings. And what is the expected format of the date?

```
     * @param fromDate if passed a date in the {@code yyyy/MM/dd} format, then only matches found on or after this date will be returned; if {@code} null or the empty string, then no lower limit on the match date is considered
     * @param toDate if passed a date in the {@code yyyy/MM/dd} format, then only matches found on or before this date will be returned; if {@code} null or the empty string, then no upper limit on the match date is considered
```

And even this is a bit incorrect, because the timestamp parsed from this date refers to the start of the date, so technically the `toDate` will match events occurring exactly at midnight, not later during the day, so the behavior isn't "on or before", but just "before".

Reply:
> because the timestamp parsed from this date refers to the start of the date

I think this is an important point, what do you think about adding 24hrs to the timestamp of the provided `toDate`?